nbs/00_utils.ipynb

```diff
@@ -0,0 +1,651 @@
+{
```

Comment:
<p>Please place this under a #| hide, e.g.</p><pre class="ql-syntax" spellcheck="false">#| hide
%load_ext autoreload
%autoreload 2
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/tsfeatures/pull/39/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>